BookOfHabits/BookOfHabits/Controllers/CardsController.cs

```diff
@@ -0,0 +1,55 @@
+﻿using AutoMapper;
+using BookOfHabits.Requests.Card;
+using BookOfHabits.Responses.Card;
+using BookOfHabitsMicroservice.Application.Models.Card;
+using BookOfHabitsMicroservice.Application.Services.Abstractions;
+using Microsoft.AspNetCore.Mvc;
+
+namespace BookOfHabits.Controllers
+{
+    [ApiController]
+    [Route("api/v1/[controller]")]
+    public class CardsController(ICardsApplicationService cardsApplicationService,
+                                 IMapper mapper) : ControllerBase
+    {
+        [HttpGet]
+        public async Task<IEnumerable<CardShortResponse>> GetAllCards()
+        {
+            IEnumerable<CardModel> persons = await cardsApplicationService.GetAllCardsAsync(HttpContext.RequestAborted);
+            return persons.Select(mapper.Map<CardShortResponse>);
+        }
+
+        [HttpGet("{id:guid}")]
+        public async Task<CardDetailedResponse> GetCardById(Guid id)
+        {
+            var card = await cardsApplicationService.GetCardByIdAsync(id, HttpContext.RequestAborted);
+            return mapper.Map<CardDetailedResponse>(card);
+        }
+
+        [HttpPost]
+        public async Task<CardShortResponse> CreateCard(CreateCardRequest request)
+        {
+            var card = await cardsApplicationService.AddCardAsync(mapper.Map<CreateCardModel>(request), HttpContext.RequestAborted);
+            return mapper.Map<CardShortResponse>(card);
+        }
+
+        [HttpPut]
+        public async Task UpdateCardAsync(UpdateCardRequest request)
+        {
```

Comment:
if we have one instruction only it would be great to use => instead of "{return ...}"

Reply:
I think it's too early to wipe your tails =)